### PR TITLE
Version number bumps for 4.6

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-1272

### CODE REVIEW

**Comment by adellape** [comment]
LGTM for OLM-related bumps. :+1:

File: modules/understanding-upgrade-channels.adoc [hunk 1]
```diff
@@ -10,14 +10,16 @@
...
-{product-title}. For instance, {product-title} {product-version}
-upgrade channels will never include an upgrade to a 4.5 release. This strategy ensures that
-administrators explicitly decide to upgrade to the next minor version of
-{product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the `openshift-install`
-binary file for a specific version of {product-title} always installs that version.
+recommending the appropriate release versions for cluster upgrade. By
+controlling the pace of upgrades, these upgrade channels allow you to choose an
+upgrade strategy. Upgrade channels are tied to a minor version of
+{product-title}. For instance, {product-title} {product-version} upgrade
+channels will never include an upgrade to a 4.6 release. This strategy ensures
```
Comment (codyhoag) [inline modules/understanding-upgrade-channels.adoc:17]: See my comment on this in another pending PR: https://github.com/openshift/openshift-docs/pull/25307#issuecomment-688885771. I believe this should be `4.7` (4.5 doc was outdated, which I think should be `4.6`). Waiting on confirmation though :slightly_smiling_face:

File: modules/cluster-logging-collector-envvar.adoc [hunk 2]
```diff
@@ -5,13 +5,8 @@
...
 = Configuring the logging collector using environment variables
 
-You can use environment variables to modify the
-configuration of the Fluentd log collector.
-
-[NOTE]
-===
-Configuring Fluentd using environment variables is deprecated in {product-title} 4.5. The process for configuring Fluentd will be different in {product-title} 4.6.
-===
+You can use environment variables to modify thec onfiguration of the Fluentd log
```
Comment (ahardin-rh) [inline modules/cluster-logging-collector-envvar.adoc:8]: the configuration

**Comment by bobfuru** [comment]
LGTM

File: metering/metering-upgrading-metering.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@ You can upgrade metering to {product-version} by updating the Metering Operator
 
 == Prerequisites
 
-*  The cluster is updated to 4.5.
+*  The cluster is updated to 4.6.
```
Comment (lbarbeevargas) [inline metering/metering-upgrading-metering.adoc:12]: Updates to _Upgrading metering_ LGTM.

**Pull request merged**